VAMobile/src/screens/SplashScreen/SplashScreen.tsx

```diff
@@ -11,19 +11,35 @@ function SplashScreen({}: SplashScreenProps) {
   const splashStyles: ViewStyle = {
     flexGrow: 1,
     justifyContent: 'center',
-    backgroundColor: theme.colors.background.splashScreen,
```

Comment:
I've not retired splashScreen as an option here as it is still used on the login Carousel which will be updated via another future ticket.